backend/README.md

```diff
@@ -28,26 +28,9 @@ cargo run
 Configuration
 -------------
 
-The backend loads configuration from `config.toml` in the current working directory.
+The backend loads configuration from a file called `config.toml` in the current working directory.
```

Comment:
Actually, since #93, we also try to load it from `/etc/tobira/config.toml`. See

https://github.com/elan-ev/tobira/blob/05f96e094ae7862b91ca0324d08f028a7749aac0/backend/server/src/config.rs#L17

Reply:
```suggestion
The backend loads its configuration from the first file found at the following locations:
- `config.toml`
- `/etc/tobira/config.toml`
```